src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,87 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: FC<Props> = ({ person }) => {
+  const { peopleSlug } = useParams();
+
+  const {
+    name,
+    sex,
+    born,
+    died,
+    fatherName,
+    motherName,
+    slug,
+  } = person;
+
+  const currentRow = slug === peopleSlug;
+  // const parentsName = (
+  //   (fatherName || motherName) ? peopleSlug?.split('-')
```

Comment:
The commented code on lines 25-28 and 30-32 should be removed if it's not being used. It's good practice to keep the codebase clean and only keep the part of the code that is being used.